portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1453,9 +1453,11 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	'getMutationProfileIds': makeCachedPromiseFunction(
 		function(self, fetch_promise) {
 		    self.getGeneticProfiles().then(function (profiles) {
-			fetch_promise.resolve(profiles.map(function (p) {
-			    return p.id;
-			}));
+			fetch_promise.resolve(
+                            profiles
+                            .filter(function (p) {return p.genetic_alteration_type == "MUTATION_EXTENDED";})
```

Comment:
In general it's better to use triple equalities in javascript, double equality does more fuzzy equalities. In this case I don't think it makes a difference but just as a rule I like to use triple equality.